RealmTasks Android/app/build.gradle

```diff
@@ -44,4 +59,11 @@ dependencies {
     compile('io.realm:android-adapters:1.3.0') {
         exclude group: "io.realm"
     }
+    // Dependency for Facebook Sign-in
+    compile('com.facebook.android:facebook-android-sdk:4.15.0')
```

Comment:
This line includes parenthesis and next line doesn't include parenthesis. Should we remove parenthesis here or add parenthesis on next line?